addon/lib/shooter.js

```diff
@@ -286,6 +287,13 @@ const ShotContext = Class({
     },
     copyLink: function () {
       clipboard.set(this.shot.viewUrl, "text");
+      notifications.notify({
+        title: "Link Copied",
+        text: "Link to the shot detail copied to the clipboard.",
+        data: "",
```

Comment:
data and onClick are optional.

Maybe the text could be "The link to your shot has been copied to the clipboard"?  Also, can we use the link icon here? addon/data/copy.png